lntest/node/config.go

```diff
@@ -207,7 +207,7 @@ func (cfg *BaseNodeConfig) GenArgs() []string {
 	nodeArgs := []string{
 		"--bitcoin.active",
 		"--nobootstrap",
-		"--debuglevel=debug",
+		"--debuglevel=debug,DISC=trace",
```

Comment:
how exactly does `DISC=trace` work here? Does it target logs coming from the `DISC` subsystem?

Reply:
yeah it will only turn on `trace` for `DISC`.